preprocessors/semanticSeg/segment.py

```diff
@@ -202,13 +196,11 @@ def run_segmentation(url,
     with torch.no_grad():
         # get segmentation results
         logging.info(
-            "running segmentation model"
-            " with start time: {}".format(str(time.time())))
+            "running segmentation model")
         scores = segmentation_module(singleton_batch,
                                      segSize=output_size)
         logging.info(
-            "run successful with"
-            " end time: {}".format(str(time.time())))
+            "run successful with")
```

Comment:
since you removed the timestamp, this no longer makes sense. suggest "finished segmentation: success" so that it is clear what is finishing